source/uri-options/tests/tls-options.yml

```diff
@@ -300,3 +596,135 @@ tests:
         hosts: ~
         auth: ~
         options: {}
+    # 4 permutations of [tlsAllowInvalidCertificates=true/false, tlsDisableOCSPEndpointCheck=true/false]
```

Comment:
Add missing tests for `tlsAllowInvalidCertificates` and `tlsDisableOCSPEndpointCheck`